post-process.py

```diff
@@ -0,0 +1,161 @@
+import os
+import pandas as pd
+from io import StringIO as sio
+import re
+import argparse
+from pathlib import Path
+import glob
+
+def parse_cli_arguments(arguments: list[str] = None,
+                        ) -> argparse.ArgumentParser:
+    """ Parses command line arguments
+
+    Reads in arguments used in the command line and passes them into the parser object.
+
+    Args:
+        None.
```

Comment:
This does appear to have an argument, not sure it actually needs one so either the argument can be removed or the Args list needs updating.